.dockerignore

```diff
@@ -9,12 +9,13 @@
 
 
 # nextjs files
-!apps/*/src/
-!apps/*/public/
-!apps/*/.next/
-!apps/*/*.ts
+!apps/
```

Comment:
The line `!apps/*/src/` looks to me like it should be sufficient to get the `apps/web/src/*` directory allowed, but it wasn't working. To resolve, I added `!apps/` and I'm calling it a day.

Reply:
I completely assumed that `!apps/*/src` would expand to include any subdirectories so great catch!